lib/plivo/xml/multipartycall.rb

```diff
@@ -0,0 +1,175 @@
+module Plivo
+  module XML
+    class MultiPartyCall < Element
+      @nestables = []
+      @valid_attributes = %w[role maxDuration maxParticipants waitMusicUrl
+                             waitMusicMethod agentHoldMusicUrl agentHoldMusicMethod
+                             customerHoldMusicUrl customerHoldMusicMethod record
+                             recordFileFormat recordingCallbackUrl recordingCallbackMethod
+                             statusCallbackEvents statusCallbackUrl statusCallbackMethod
+                             stayAlone coachMode mute hold startMpcOnEnter endMpcOnExit
+                             enterSound enterSoundMethod exitSound exitSoundMethod
+                             onExitActionUrl onExitActionMethod relayDTMFInputs]
+
+      VALID_ROLE_VALUES = %w[agent supervisor customer]
+      VALID_MAX_DURATION_VALUES = (300..28800).to_a
+      VALID_MAX_PARTICIPANT_VALUES = (2..10).to_a
+      VALID_WAIT_MUSIC_METHOD_VALUES = %w[GET POST]
+      VALID_AGENT_HOLD_MUSIC_METHOD_VALUES = %w[GET POST]
+      VALID_CUSTOMER_HOLD_MUSIC_METHOD_VALUES = %w[GET POST]
```

Comment:
all `%w[GET POST]` can be clubbed under single variable `VALID_METHODS`
we do not need multiple variables with same value